attributecert/attributecert.go

```diff
@@ -336,11 +336,11 @@ type CommonCriteriaMeasures struct {
 }
 
 type TBBSecurityAssertions struct {
-	Version          int
+	Version          int                    `asn1:"optional,default=0"`
```

Comment:
I think this should have to be ':' not '='

Also ints already default to 0. If you mark optional, is that enoguh?

Reply:
I'm still seeing "default=0" in my diff. Is this intentional? :)